src/runtime/auto-prompt-manager-test.js

```diff
@@ -1076,6 +1085,77 @@ describes.realWin('AutoPromptManager', {}, (env) => {
     expect(alternatePromptSpy).to.not.be.called;
   });
 
+  it('should log events when a large prompt overrides the miniprompt', async () => {
+    win./*OK*/ innerWidth = 500;
+    setExperiment(win, ExperimentFlags.DISABLE_DESKTOP_MINIPROMPT, true);
+    const expectedEvent = {
+      eventType: AnalyticsEvent.EVENT_DISABLE_MINIPROMPT_DESKTOP,
+      eventOriginator: EventOriginator.SWG_CLIENT,
+      isFromUserAction: false,
+      additionalParameters: {
+        publicationid: pubId,
+        promptType: AutoPromptType.CONTRIBUTION,
+      },
+    };
+
+    await autoPromptManager.showAutoPrompt({
+      autoPromptType: AutoPromptType.CONTRIBUTION,
+      alwaysShow: true,
+      displayLargePromptFn: alternatePromptSpy,
+    });
+    expect(logEventSpy).to.be.calledWith(expectedEvent);
```

Comment:
I think there is a calledOnceWith method you can use instead, and remove the next line.

Reply:
Good call! I've combined those